doc/api/n-api.md

```diff
@@ -3149,6 +3149,26 @@ defined in
 [Section 7.2.14](https://tc39.github.io/ecma262/#sec-strict-equality-comparison)
 of the ECMAScript Language Specification.
 
+#### napi_arraybuffer_detach
+<!-- YAML
+added: REPLACEME
+-->
+
+```C
+napi_status napi_arraybuffer_detach(napi_env env,
+                                    napi_value arraybuffer)
+```
+
+* `[in] env`: The environment that the API is invoked under.
+* `[in] arraybuffer`: The JavaScript value of arraybuffer to be detached.
+
+Returns `napi_ok` if the API succeeded.
+
+This API represents the invocation of the ArrayBuffer Detach operation as
+defined in
+[Section 24.1.1.3](https://tc39.es/ecma262/#sec-detacharraybuffer)
```

Comment:
should probably just say "defined in ecma262", no guarantee of those numbers staying constant.

Reply:
We already have lots of ecma section references in `n-api.md`. IMO it might be helpful to jump between the spec and n-api document since the detailed operation spec would not be listed in `n-api.md`. If there is any permanent link to ecma spec sections would be better. Or we could constantly update the links.